OCR/frontend/dev-dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:20.17.0-alpine3.20
+
+RUN apk add --update nodejs npm
+
+
+COPY . /app
+
+WORKDIR /app
+RUN npm install 
+
+
+# CMD npm run dev -- --host
+CMD vite
```

Comment:
what does the `vite` cmd do? is it different from the command in the docker compose yaml

Reply:
Oops, great catch - in the compose I overwrite the command so I never really tested it without compose running it. Just updated it to be correct so you can run the just this one instance if you want instead without the compose (not that I'd suggest it)